src/test/java/com/rultor/agents/github/qtn/QnAskedByTest.java

```diff
@@ -78,4 +79,40 @@ public void excludesRultorFromListOfCommanders() throws Exception {
             )
         );
     }
+
+    /**
+     * QnAskedBy can include all architects in the list of commanders.
+     * @throws Exception In case of error.
+     */
+    @Test
+    public void includesArchitectsInListOfCommanders() throws Exception {
+        final MkGithub github = new MkGithub();
+        final Repo repo = github.randomRepo();
+        repo.collaborators().add("testuser2");
+        final Issue issue = repo.issues().create("needs", "deployment");
+        final String action = "deploy";
+        issue.comments().post(action);
+        final Comment.Smart comment = new Comment.Smart(
+            issue.comments().get(1)
+        );
+        final Question question = Mockito.mock(Question.class);
+        final QnAskedBy qab = new QnAskedBy(
+            new Profile.Fixed(
+                new XMLDocument(
+                    String.format(
+                        "<p><entry key='architect'>%s</entry></p>",
+                        repo.github().users().self().login()
+                    )
+                )
+            ),
+            String.format(
+                "/p/entry[@key='%s']/entry[@key='commanders']/item/text()",
```

Comment:
@original-brownbear ok and the test fails without the change in `QnAskedBy`, correct?